main/ZgatewayLORA.ino

```diff
@@ -69,16 +185,38 @@ void LORAtoMQTT() {
     taskMessage = taskMessage + xPortGetCoreID();
     //trc(taskMessage);
 #  endif
-    String packet;
-    packet = "";
+    // Create packet and reserve null terminator space
+    byte packet[packetSize + 1];
+    boolean binary = false;
     for (int i = 0; i < packetSize; i++) {
-      packet += (char)LoRa.read();
+      packet[i] = (char)LoRa.read();
+
+      if (packet[i] < 32 || packet[i] > 127)
+        binary = true;
```

Comment:
I'm not getting this criterion about identifying a binary depending on the <32 or >127 could you explain ?